include/reactor-uc/network_channel.h

```diff
@@ -31,14 +31,27 @@ typedef enum {
 typedef enum {
   NETWORK_CHANNEL_TYPE_TCP_IP,
   NETWORK_CHANNEL_TYPE_COAP_UDP_IP,
+  NETWORK_CHANNEL_TYPE_I2C,
```

Comment:
One thing that will make I2C challenging is that we currently require channels to be two-way. This is needed even if there is only a connection going one way. The reason is that we distribute the start tags by forwarding it on all channels. If we only could forward it on outgoing connections we would need to carefully pick the federate selecting the start tag.